### PR TITLE
Custom error message

### LINKED ISSUE #104: Add customised error message for add command

### PR DESCRIPTION
**Pull request by antonlee59**
close #104

### CODE REVIEW

File: src/main/java/seedu/ultron/logic/parser/AddCommandParser.java [hunk 1]
```diff
@@ -63,5 +81,21 @@ public AddCommand parse(String args) throws ParseException {
     private static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
         return Stream.of(prefixes).allMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
     }
+    private static boolean arePrefixesAbsent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
```
Comment (Lanreath) [inline src/main/java/seedu/ultron/logic/parser/AddCommandParser.java:84]: Duplicate method

File: src/main/java/seedu/ultron/logic/parser/AddCommandParser.java [hunk 2]
```diff
@@ -39,8 +39,26 @@ public AddCommand parse(String args) throws ParseException {
                 ArgumentTokenizer.tokenize(args, PREFIX_POSITION, PREFIX_COMPANY, PREFIX_EMAIL,
                         PREFIX_STATUS, PREFIX_REMARK, PREFIX_KEYDATE);
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_POSITION, PREFIX_COMPANY, PREFIX_EMAIL, PREFIX_STATUS)
-                || !argMultimap.getPreamble().isEmpty()) {
+        // if (!arePrefixesPresent(argMultimap, PREFIX_POSITION, PREFIX_COMPANY, PREFIX_EMAIL, PREFIX_STATUS)
```
Comment (Lanreath) [inline src/main/java/seedu/ultron/logic/parser/AddCommandParser.java:42]: Clear this comment

**Review by Lanreath — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/ultron/logic/parser/AddCommandParser.java (hunk 1 above)
Comment (antonlee59) [reply src/main/java/seedu/ultron/logic/parser/AddCommandParser.java:84]: It's not duplicate, the latter is checking whether all the fields are absent and the former is checking whether there are any absent

**Review by Lanreath — APPROVED** [review]
LGTM

**Pull request merged**